addons/website_event_track/views/event_track_templates_page.xml

```diff
@@ -174,6 +174,7 @@
                 class="tab-content collapse d-md-block o_wesession_track_aside_tabs">
                 <div class="tab-pane fade show active" id="track_list" role="tabpanel">
                     <ul class="list-unstyled mb-0">
+                        <t t-set="user_event_manager" t-value="env.user.has_group('event.group_event_manager')"/>
```

Comment:
Idea was more to add it in the controller that renders those templates ... that way it does not require to update the view and is available directly after pulling the code.

Reply:
Indeed, that's better, thanks. I have done the change.